src/ui/components/logo.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
don't use `eslint-disable` use `import React from 'react'` follow this [docs](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md)